docs/dev-guide/android-sdk.md

```diff
@@ -370,68 +371,90 @@ Please see `JitsiMeetActivity`, which registers for all the events and can serve
 
-##### VIDEO_MUTED_CHANGED
-Broadcasted when videoMuted state changed.
-The `data` HashMap contains a `muted` key with state of the videoMuted for the localParticipant.
+Broadcasted when the local participant's audio is muted or unmuted. `data` contains the following information:
+
+- `muted`: a boolean indicating whether the audio is muted or not.
 
 ##### PARTICIPANT_JOINED
-Broadcasted when a participant has joined the conference.
-The `data` HashMap contains information of the participant that has joined.
-Depending of whether the participant is the local one or not, some of them are 
-present/missing.
-    isLocal
-    email
-    name
-    participantId
+
+Broadcasted when a participant has joined the conference. `data` contains the following information:
+
+- `email`: the email of the participant.
+- `name`: the name of the participant.
+- `role`: the role of the participant.
+- `participantId`: the id of the participant.
 
 ##### PARTICIPANT_LEFT
-Broadcasted when a participant has joined the conference.
-The `data` HashMap contains information of the participant that has left.
-Depending of whether the participant is the local one or not, some of them are 
-present/missing.
-    isLocal
-    email
-    name
-    participantId
+
+Called when a participant has left the conference. `data` contains the following information:
+
+- `participantId`: the id of the participant that left.
 
 ##### ENDPOINT_TEXT_MESSAGE_RECEIVED
-Broadcasted when an endpoint text message is received.
-The `data` HashMap contains a `senderId` key with the participantId of the sender and a `message` key with the content.
+
+Broadcasted when an endpoint text message is received. The `data` HashMap contains a `senderId` key with the
+participantId of the sender and a `message` key with the content.
+
+#### SCREEN_SHARE_TOGGLED
+
+Broadcasted when a participant starts or stops sharing his screen. `data` contains the following information:
+
+- `participantId`: Id of the participant that started or stopped sharing his screen.
+- `sharing`: True if the participant is sharing his screen, false otherwise.
 
 ##### PARTICIPANTS_INFO_RETRIEVED
-Broadcasted when a RETRIEVE_PARTICIPANTS_INFO action is called.
-The `data` HashMap contains a `participantsInfo` key with a list of participants information and a `requestId` key with the id that was sent in the RETRIEVE_PARTICIPANTS_INFO action.
+
+Broadcasted when a RETRIEVE_PARTICIPANTS_INFO action is called. The `data` HashMap contains a `participantsInfo` key
+with a list of participants information and a `requestId` key with the id that was sent in the
+RETRIEVE_PARTICIPANTS_INFO action.
 
 ##### CHAT_MESSAGE_RECEIVED
-Broadcasted when a chat text message is received.
-The `data` HashMap contains a `senderId` key with the participantId of the sender, a `message` key with the content, a `isPrivate` key with a boolean value and a `timestamp` key.
```

Comment:
The `timestamp` key never existed in the example I tried. Thus, I removed it.

Reply:
It's optional, it may or may not be there, please leave it there.